src/bisection.test.ts

```diff
@@ -94,19 +90,22 @@ test('manual tri-section', () => {
     3
   );
 
-  cm.split(
-    witness(incorrectStates, 3),
-    fingerprints(incorrectStates, [4, 5, 6]),
-    witness(correctStates, 6),
-    proposerId
-  );
-  expect(cm.detectFraud(witness(incorrectStates, 4), {root: 4}, witness(incorrectStates, 5))).toBe(
-    true
-  );
+  const consensusWitness = generateWitnessFromHashes(cm.stateHashes, 1);
+  const disputeWitness = generateWitnessFromHashes(cm.stateHashes, 2);
+
+  cm.split(consensusWitness, fingerprints(incorrectStates, [4, 5, 7]), disputeWitness, proposerId);
+
+  expect(
+    cm.detectFraud(
+      generateWitnessFromHashes(cm.stateHashes, 1),
+      {root: 4},
+      generateWitnessFromHashes(cm.stateHashes, 2)
+    )
+  ).toBe(true);
 });
 
-const amountOfStates = 90;
-const maxSplits = 90;
+const amountOfStates = 25;
```

Comment:
Any particular reason to reduce this test case from 90 to 25?

Reply:
Now that we're generating merkle proofs the fuzzy test can take a bit longer if we have a lot of elements.